feature/experimental/p4rt/otg_tests/traceroute_packetout_test/rundata_test.go

```diff
@@ -0,0 +1,10 @@
+// Code generated by go run tools/addrundata; DO NOT EDIT.
+package traceroute_packetout_test
+
+import "github.com/openconfig/featureprofiles/internal/rundata"
+
+func init() {
```

Comment:
Do we need this file in this PR ?

Reply:
Checked with the team. Looks like this test was added in PR908, but the rundata file was not included in that PR. and w/o this rundata_test file, this PR is failing the checks required to merge.